src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -89,11 +99,13 @@ public boolean removeByInputsAndOutput(FluidStack output, FluidStack... inputs)
         return false;
     }
 
+    @MethodDescription(description = "groovyscript.wiki.removeAll", priority = 2000, example = @Example(commented = true))
```

Comment:
if you merge `master` in, you can get rid of the very repetitive `description = "groovyscript.wiki.[methodname]"` stuff

Reply:
uhhh idk how